headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/resource/v1_0/WarehouseItemResourceImpl.java

```diff
@@ -232,6 +234,15 @@ public WarehouseItem postWarehousIdWarehouseItem(
 					getCommerceInventoryWarehouseItemId()));
 	}
 
+	private Date _addDaysToDate(Date date, int increment) {
+		Calendar cal = Calendar.getInstance();
+
+		cal.setTime(date);
```

Comment:
Ideally there'd be a newline after `cal.setTime(date);` -- we use newlines as a [signal that order matters](https://docs.google.com/document/d/188YFP0V4n9WJIPYZL3jGBRBvEsvBNmHOU97Fdd_g6Ew/edit#heading=h.pccy0tblj4xd), and it definitely matters here that `setTime` comes before `add`.